### DISCUSSION TITLE
TypeScript Config Path not working on Next 13

### DISCUSSION THREAD

**Discussion by bnhr** [vercel/next.js#43664] [Help]
Hi folks, I'm having trouble using path and baseUrl inside the Next 13 project, it seems it's not working as expected.
Here's my config:

```
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "baseUrl": ".",
    "paths": {
      "@/source/*": ["source/*"]
    }
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
  "exclude": ["node_modules"]
}
```

All of the directories are wrapped inside the src folder and this is not working, I tries to renamed it to source folder, and still not working. Is this issue with Next or TypeScript?

**Comment by GabenGar** [comment]
There is no mention of `src` folder in your config.

**Reply by bnhr** [reply]
All of my folder (pages, component, utils, etc) are inside folder named source. It's src before, and I renamed it

**Reply by GabenGar** [reply]
NextJS doesn't let you put `pages` folder willy-nilly. You either have it at top level or in `src` folder.

**Reply by bnhr** [reply]
I see, I tried to install new fresh Next 13, but still can't import from absolute path
